src/content/docs/linter/index.mdx

```diff
@@ -4,180 +4,205 @@ description: How to use the Biome linter.
 ---
 
 import NumberOfRules from "@/components/generated/NumberOfRules.astro";
-import RecommendedRules from "@/components/generated/RecommendedRules.astro";
+import PackageManagerBiomeCommand from "@/components/PackageManagerBiomeCommand.astro";
 
-Biome's linter statically analyzes your code to catch common errors and to help writing idiomatic code.
+Biome's linter statically analyzes your code to find and fix common errors and to help you write better code.
+It [supports multiple languages](/internals/language-support) and provides a total of [**<NumberOfRules/> rules**](/linter/rules/).
 
-<NumberOfRules />
 
-## Use the linter via CLI
+## CLI
 
-You can start by running the CLI to check for possible errors using the following command:
+The following command runs the linter on all files in the `src` directory:
 
-```shell
-biome lint ./src
-```
-
-For more information about all the available options, check the [CLI page](/reference/cli#biome-lint)
+<PackageManagerBiomeCommand command="lint ./src" />
 
-## Rule pillars
-
-We believe that rules should be informative and explain to the user why a rule is triggered and tell the user what they should to do fix the error. A rule should follow these **pillars**:
-1. Explain to the user the error. Generally, this is the message of the diagnostic.
-2. Explain to the user **why** the error is triggered. Generally, this is implemented with an additional node.
-3. Tell the user what they should do. Generally, this is implemented using a code action. If a code action is not applicable a note should tell the user what they should do to fix the error.
+Note that if you pass a glob as a parameter, it will be expanded by your shell.
+The result of the expansion depends on your shell and its configuration.
+For example, some shells don't support the recursive glob `**` or the alternation `{}` in the following command:
```

Comment:
```suggestion
Note that if you pass a glob as a parameter, your shell will expand it.
The result of the expansion depends on your shell and its configuration.
For example, some shells don't support the recursive glob `**` or the alternation `{}` in the following command:
```

We have the same phrase in the formatter. Maybe we should have it in a common section, otherwise we will struggle to update both.

Reply:
I will try to address the duplication concern in a future PR.